include/9on12PipelineStateStructures.h

```diff
@@ -333,7 +334,15 @@ namespace D3D9on12
             }
         }
         rasterizerDesc.DepthBiasClamp = 0.0f;
-        rasterizerDesc.DepthClipEnable = rasterizerID.DepthClipEnable ? TRUE : FALSE;
+
+        if (rasterizerID.DrawingPreTransformedVertices && !bDepthEnabledAndBound)

```

Comment:
I'm not sure we should have the depth enabled check here. If the app is using a `<` or `<=` depth test, then the depth test will end up removing pixels that fall out of range. If they're using `>` or `>=`, then I think these pre-transformed and pre-clipped vertices *shouldn't* be depth clipped, and instead will just be depth clamped.

Reply:
The behavior that I was seeing before adding the check for bDepthEnabledAndBound was that the test was always drawing the triangle, regardless of whether ZENABLE was true or false. Depth Compare Function is set to `<=` for these tests, so I was also surprised that it was still drawing.

Reply:
Talked with Jesse offline, we think this should be fine as is.